core/src/main/resources/logback.xml

```diff
@@ -29,7 +29,7 @@
     </root>
     <logger name="org.eclipse.jetty" level="INFO"/>
     <logger name="io.netty" level="INFO"/>
-    <logger name="Exposed" level="DEBUG"/> <!-- Log SQL statements executed by Exposed -->
+    <logger name="Exposed" level="TRACE"/> <!-- Log SQL statements executed by Exposed -->
```

Comment:
I think there are some more `logback.xml` files in other modules that would need to be adapted as well.